src/CoreFx.Private.TestUtilities/src/CoreFx.Private.TestUtilities.csproj

```diff
@@ -29,6 +29,9 @@
     <Compile Include="$(CommonPath)\System\PasteArguments.cs">
       <Link>Common\System\PasteArguments.cs</Link>
     </Compile>
+    <Compile Condition="'$(TargetGroup)' == 'uap'" Include="$(CommonTestPath)\System\Threading\ThreadTestHelpers.cs">
```

Comment:
you need to add this class to the reference assembly https://github.com/dotnet/corefx/blob/master/src/CoreFx.Private.TestUtilities/ref/CoreFx.Private.TestUtilities.cs